app/src/main/res/values/strings.xml

```diff
@@ -20,17 +20,17 @@
     <!-- Long click options -->
     <string name="long_click_title">What do you want to do?</string>
     <string name="long_click_open">Open \"%1s\"</string>
-    <string name="long_click_store">See its store page</string>
+    <string name="long_click_store">View store page</string>
     <string name="long_click_settings">Access its settings</string>

```

Comment:
As per the comment about store page, should we also change this one?
```suggestion
    <string name="long_click_settings">Access settings</string>
```

Reply:
No, because it could be general settings.
Very easy to get it wrong in translation thinking it is the app settings or somesuch.

Reply:
Ok you are right, we let it like this then.